src/components/AppLayout/utils.ts

```diff
@@ -1,8 +1,7 @@
+import { IMenuItem } from "@saleor/macaw-ui";
```

Comment:
We should be getting rid off the `I` prefixed interfaces whenever we find the chance

Reply:
Will do :+1:

Reply:
Since it's already merged I created ticket to fix it later
https://app.clickup.com/t/mh6qyx